src/connector/connector_utils.go

```diff
@@ -948,3 +948,26 @@ func RemoveRoCEDevice(device string) ([]string, string, error) {
 
 	return devices, multiPathName, nil
 }
+
+// RemoveNvmeFcDevice remove dm device if present
+func RemoveNvmeFcDevice(device string) (string, error) {
+	var multiPathName string
+	var err error
+	if strings.HasPrefix(device, "dm") {
+		multiPathName = device
+		// devices: nvme0n1, nvme2n1,
+		_, err = getDeviceFromDM(multiPathName)
+		if err != nil {
+			log.Warningf("Get the devices from the multipath %s error: %v", multiPathName, err)
```

Comment:
As this is just warning, this means the sys class has issues and we are ignoring it. Further when you try to flush the device without having the sysfs entries, will it create issues?